melosys-eessi-test/src/test/java/no/nav/melosys/eessi/KafkaAdminTjenesteTestIT.java

```diff
@@ -82,10 +82,10 @@ void stoppOgStartConsumer_consumerStoppedOgStartes() {
         when(oppgaveConsumer.hentOppgave(oppgaveID2)).thenReturn(oppgaveDto2);
         when(oppgaveConsumer.hentOppgave(oppgaveID3)).thenReturn(oppgaveDto3);
 
-        kafkaTemplate.send(lagOppgaveIdentifisertRecord(oppgaveID, FNR, "1", rinaSaksnummer)).get();
-        kafkaTemplate.send(lagOppgaveIdentifisertRecord(oppgaveID1, FNR, "1", rinaSaksnummer)).get();
-        kafkaTemplate.send(lagOppgaveIdentifisertRecord(oppgaveID2, FNR, "1", rinaSaksnummer)).get();
-        kafkaTemplate.send(lagOppgaveIdentifisertRecord(oppgaveID3, FNR, "1", rinaSaksnummer)).get();
+        kafkaTemplate.send(lagOppgaveIdentifisertRecord(oppgaveID, "1", rinaSaksnummer)).get();
```

Comment:
Når jeg leser denne koden fra Github, har jeg ikke peiling hva "1" er for noe. Hva representerer "1"? :smile:

Reply:
Versjonsnummer for oppgaven.